ckan/ckanext/ckanext-ytp_main/ckanext/ytp/templates/scheming/package/snippets/additional_info.html

```diff
@@ -41,7 +41,7 @@
     <tr>
       <th>{{ _('Maintainer') }}</th>
       <td>
-        <a href="/data/{{ h.lang() }}/organization/{{ pkg_dict.organization.get('name') }}">{{ pkg_dict.organization.get('name') }}</a>
+        <a href="/data/{{ h.lang() }}/organization/{{ pkg_dict.organization.get('name') }}">{{ h.extra_translation(pkg_dict.organization, 'title') }}</a>
```

Comment:
```suggestion
        <a href="/data/{{ h.lang() }}/organization/{{ pkg_dict.organization.get('name') }}">{{ h.get_translated(pkg_dict.organization, 'title') }}</a>
```

`extra_translation` helper is old helper from a time ckan core did not have `get_translated`, all the new code should use `get_translated` instead.